wgpu-hal/src/metal/adapter.rs

```diff
@@ -850,20 +850,26 @@ impl super::PrivateCapabilities {
             } else {
                 Self::version_at_least(major, minor, 11, 0)
             },
+            //Depth clipping is supported on all macOS GPU families and iOS family 4 and later
+            supports_depth_clamping: device.supports_feature_set(MTLFeatureSet::iOS_GPUFamily4_v1)
+            || device.supports_feature_set(MTLFeatureSet::macOS_GPUFamily1_v1)
         }
     }
 
     pub fn features(&self) -> wgt::Features {
         use wgt::Features as F;
 
         let mut features = F::empty()
-            | F::DEPTH_CLAMPING
             | F::TEXTURE_COMPRESSION_BC
             | F::MAPPABLE_PRIMARY_BUFFERS
             | F::VERTEX_WRITABLE_STORAGE
             | F::TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES
             | F::POLYGON_MODE_LINE
             | F::CLEAR_COMMANDS;
+        
+        if self.supports_depth_clamping {
+            features |= F::DEPTH_CLAMPING;
```

Comment:
nit: see how this is done for other flags, i.e.
```rust
features.set(..., self.supports_depth_clamping);
```

Reply:
(facepalm).
Will do

Reply:
Done